packages/turf-boolean-within/index.js

```diff
@@ -34,6 +34,7 @@ function booleanWithin(feature1, feature2) {
         case 'LineString':
             return booleanPointOnLine(geom1, geom2, {ignoreEndVertices: true});
         case 'Polygon':
+        case 'MultiPolygon':
             return booleanPointInPolygon(geom1, geom2, {ignoreBoundary: true});
```

Comment:
Don't you love easy fixes :)

Reply:
Very easy fix 👍 Hence why I didn't tackle `MultiLineStrings` or anything more complicated than that, lol